Studio/ConnecttoaNeptuneinstance.md

```diff
@@ -90,6 +90,10 @@ Press “review and launch”, then in the dialog, create and download key pair:
 
 &nbsp;
 
+### Ensure you have the proper IAM acces rights to Neptune infrastructure
```

Comment:
Until we change the Neptune connection form to only require the cluster endpoint url the user needs to have AWS access keys and this specific IAM action policy